### DISCUSSION TITLE
Sort by district on VAC Meeting List

### DISCUSSION THREAD

**Discussion by Em-VienaTechCoordinator** [code4recovery/12-step-meeting-list#1479] [Q&A]
Hi all! 

So, a fellow GSR pointed out that the VAC meeting list - https://aavirginia.org/meetings/ - used to let you sort by district but doesn't anymore. Did that feature move? Am I looking in the wrong place? If it was removed, is that a feature we can consider re-adding? I know GSRs have mentioned its usefulness for tracking meetings in their district and putting a little order to the chaos. Many thanks for pointing me in the right direction! 

Best, 
E

**Comment by joshreisner** [comment]
hi, yes, the "Switch to Districts" mode came out of the UI because only a small subset of overall users know what districts are, however the functionality is still there if you use the search. for example:

https://aavirginia.org/meetings/?search=%22District+39%22

one approach the web servant could take to add back this feature without searching would be to add links to this existing page which has information about the districts: https://aavirginia.org/member-services/virginia-area-districts/, for example:

```
<a href="/meetings/?search=%22District 12 - Newport News%22">View Meetings in District 12</a>
```

<img width="439" alt="Screenshot 2024-08-03 at 12 34 09 PM" src="https://github.com/user-attachments/assets/90918220-5809-42e0-b5d7-8366e9ecb17c">

**Comment by Em-VienaTechCoordinator** [comment]
Ooo! Didn’t know that. Thank you!

**Comment by Em-VienaTechCoordinator** [comment]
How can we add links to that existing page?

**Reply by joshreisner** [reply]
i'm assuming the area webservant can just edit the page in wordpress - not sure how they have it set up

**Comment by kiyote33** [comment]
Hi @Em-VienaTechCoordinator, Note that Districts are still available in the Legacy UI, which can be found on the Settings page of TSML. In Edmonton, we load the Legacy UI (with Districts enabled) by default on the Meetings page, and then also use the [tsml_ui] shortcode  on a separate "Find a meeting" page". You can check out https://edmontonaa.org to see if something like that will work for you...

**Comment by Em-VienaTechCoordinator** [comment]
Thank you!
